po/pt_BR.po

```diff
@@ -419,7 +437,7 @@ msgstr "Criar um pacote sólido"
 #, fuzzy
 #| msgid "Create a solid archive"
 msgid "Create, extract and modify archives"
-msgstr "Criar um pacote sólido"
+msgstr "O ‘Xarchiver’ é um ‘gerenciador de pacotes’, que também é conhecido por ‘gerenciador de compactação’ ou ‘compactador e descompactador de arquivos’, que permite criar, adicionar, extrair, remover e modificar os arquivos e pastas nos pacotes de vários formatos (7z, ar, arj, bzip2, gzip, lha, lzma, lz4, lzip, lrzip, lzop, rar, tar, tar.bz2, tar.gz, tar.lzma, xz, zip, zstd, deb e rpm), trata-se de um programa leve que é baseado na linguagem de programação GTK+ e é independente do tipo do ambiente de área de trabalho para a manipulação dos arquivos"
```

Comment:
I'm sure you know that this is a tooltip for a menu item, because we've had this discussion before. Why do you think you have to translate this with some kind of man page?!